CHANGELOG.md

```diff
@@ -35,6 +35,8 @@ straightforward as possible.
 ### Fixed
 - mobile upload on Android
 - device log. new format support [#565]
+- Log in by key uploading UI issue
+- Mobile view. Sync mode cover chat options
```

Comment:
Please specify github issue number in []
And add it to PR header as well

Reply:
done